requirements_rtd.txt

```diff
@@ -1,12 +0,0 @@
-configparser==3.5.0b2
```

Comment:
Why was this file removed?

Reply:
Because this file is only required for `Python 2.7`.

Reply:
I'm not sure what this file was for, I guess, to begin with. I thought it was related to the documentation (`rtd` -> `ReadTheDocs`?). Why did we need it even for Python 2.7? Anyway, I don't think it matters.

Reply:
ReadTheDocs didn't support Python 3 when we started SKLL so we needed a 2.7 requirements files.